Content.Server/GameObjects/EntitySystems/RadioSystem.cs

```diff
@@ -11,38 +13,18 @@ namespace Content.Server.GameObjects.EntitySystems
 {
     class RadioSystem : EntitySystem
     {
-        private List<RadioComponent> _activeRadios;
+        private IEntityQuery RadioQuery;
         private List<string> _messages;
 
         public override void Initialize()
         {
             base.Initialize();
 
-            _activeRadios = new List<RadioComponent>();
+            RadioQuery = new TypeEntityQuery(typeof(RadioComponent));
```

Comment:
`EntitySystems` already have an `IEntityQuery EntityQuery` use that instead of `RadioQuery`.